packages/react-native-tab-view/src/TabBar.tsx

```diff
@@ -545,6 +544,7 @@ export function TabBar<T extends Route>({
         style: tabStyle,
         defaultTabWidth,
         android_ripple,
+        ...rest,
```

Comment:
This should stay at the top. It shouldn't be possible to override the other properties due to any options that user may specify as it could break the component.

If there are specific properties that need to be at the end, then those properties should be explicitly specified, though there's probably a different issue if the other properties specified here aren't correct.